<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>copyEngineOptions</class>
 <widget class="QWidget" name="copyEngineOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>504</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="page_trasnfer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>364</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Transfer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="3" column="2">
        <widget class="QCheckBox" name="moveTheWholeFolder"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Move the whole folder</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Transfer the file rights</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="doRightTransfer"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Keep the file date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="keepDate"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Auto start the transfer</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="autoStart"/>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>278</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_19">
         <property name="toolTip">
          <string>Less performance if checked</string>
         </property>
         <property name="text">
          <string>Follow the strict order</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="followTheStrictOrder">
         <property name="toolTip">
          <string>Less performance if checked</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_error_collision">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>364</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Error and collision</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>When folder error</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="checkBoxDestinationFolderExists"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>When file error</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBoxFileError">
         <item>
          <property name="text">
           <string notr="true">Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Skip</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Put at the end</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>When file collision</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="comboBoxFileCollision">
         <item>
          <property name="text">
           <string notr="true">Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Skip</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Overwrite</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Overwrite if different</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Overwrite if newer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Overwrite if older</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Rename</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboBoxFolderError">
         <item>
          <property name="text">
           <string notr="true">Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Skip</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>When folder collision</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="comboBoxFolderCollision">
         <item>
          <property name="text">
           <string notr="true">Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Merge</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Skip</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Rename</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Check if destination folder exists</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="renamingRules">
         <property name="text">
          <string>Renaming rules</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>193</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Delete partially transferred files</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="deletePartiallyTransferredFiles"/>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="renameTheOriginalDestination"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Rename the original distination</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>label_12</zorder>
      <zorder>label_13</zorder>
      <zorder>label_6</zorder>
      <zorder>label_7</zorder>
      <zorder>label_8</zorder>
      <zorder>comboBoxFileError</zorder>
      <zorder>comboBoxFileCollision</zorder>
      <zorder>comboBoxFolderError</zorder>
      <zorder>comboBoxFolderCollision</zorder>
      <zorder>checkBoxDestinationFolderExists</zorder>
      <zorder>renamingRules</zorder>
      <zorder>label_20</zorder>
      <zorder>deletePartiallyTransferredFiles</zorder>
      <zorder>renameTheOriginalDestination</zorder>
      <zorder>label_22</zorder>
     </widget>
     <widget class="QWidget" name="page_control">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>172</width>
        <height>119</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Control</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Checksum</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QCheckBox" name="checksumOnlyOnError"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Only after error</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Ignore if impossible</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checksumIgnoreIfImpossible"/>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="doChecksum">
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Do checksum</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>242</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_performance">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>332</width>
        <height>216</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Performance</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Parallel buffer</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="sequentialBuffer">
         <property name="suffix">
          <string>KB</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="blockSize">
         <property name="suffix">
          <string>KB</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="parallelBuffer">
         <property name="suffix">
          <string>KB</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="osBuffer"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Block size</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Sequential buffer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Enable OS buffer</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="osBufferLimit">
         <property name="suffix">
          <string>KB</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2048</number>
         </property>
         <property name="value">
          <number>512</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="osBufferLimited">
         <property name="text">
          <string>OS buffer only if smaller than</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Transfer algorithm</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="transferAlgorithm">
         <item>
          <property name="text">
           <string notr="true">Automatic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Sequential</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Parallel</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>197</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Parallelize if smaller than</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="parallelizeIfSmallerThan">
         <property name="suffix">
          <string>KB</string>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="inodeThreads">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Inode threads</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_misc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>364</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Misc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Check the disk space</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>353</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="filters">
         <property name="text">
          <string>Filters</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>352</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>159</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="checkDiskSpace"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>doChecksum</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_9</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doChecksum</sender>
   <signal>clicked(bool)</signal>
   <receiver>checksumOnlyOnError</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doChecksum</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_10</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doChecksum</sender>
   <signal>clicked(bool)</signal>
   <receiver>checksumIgnoreIfImpossible</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
